### DISCUSSION TITLE
Highlights for your issues

### DISCUSSION THREAD

**Discussion by BekahHW** [open-sauced/feedback#6] [Feature request]
It would be nice to see the ability to add user-created issues to highlights.

This would allow for more users--if someone doesn't code, they can still create an issue--and allow users to showcase their communication, understanding of a codebase, debugging/problem-solving skills, and to potentially find a collaborative partner for that issue.

**Comment by 0-vortex** [comment]
Considering PRs are very similar to Issues in the GitHub API, this would be the least effort to implement among "additional URLs" types of highlights! 💯 🚀 🍕

**Comment by ghost** [comment]
> It would be nice to see the ability to add user-created issues to highlights.
> 
> This would allow for more users--if someone doesn't code, they can still create an issue--and allow users to showcase their communication, understanding of a codebase, debugging/problem-solving skills, and to potentially find a collaborative partner for that issue.

This is a very good idea and I was thinking of the same. It gives motivation to the non-code contributors.
